MAUI/DataForm/localization.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: Localization in .NET MAUI DataForm control | Syncfusion
+description: Learn about the Localization support in Syncfusion .NET MAUI DataForm(SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Localization in .NET MAUI DataForm (SfDataForm)
+
+Localization is the process of translating the application resources into different language for the specific cultures. The `SfDataForm` can be localized by adding `resource` file. 
+
+You can localize the DataFormItem [Display](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute?view=net-7.0) attribute values and validation ([Required](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.requiredattribute?view=netframework-4.8), [StringLength](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.stringlengthattribute?view=netframework-4.8) ) attributes values by using `ResourceType` display attribute or using the `GenerateDataFormItem` event.
```

Comment:
You can localize the `DataFormViewItem` [DisplayAttribute](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute?view=net-7.0) values and validation attributes ([Required](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.requiredattribute?view=netframework-4.8), [StringLength](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.stringlengthattribute?view=netframework-4.8) ) values by using `ResourceType` property of the attribute or using the `GenerateDataFormItem` event.

Reply:
content changed